CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ New configuration to access the database:
 - `EDC_SERVER_DB_CONNECTION_POOL_SIZE`
   - The property controls the maximum size that the pool is allowed to reach, including both idle and in-use connections. Basically this value will determine the maximum number of actual connections to the database backend.
   - Defaults to `3`
+- `MY_EDC_DATASOURCE_PLACEHOLDER_BASEURL`
```

Comment:
the launchers/.env.connector should correctly set this environment variable for both the CE (wrt how we want our customers to deploy it) and the EE (wrt how we deploy it)

if we do that, then we do not have to document this here as it has the proper default

Reply:
I can propose a default, but that still doesn't make it reliable. The URL as it will be seen, I think, depends on their setup.
I can default it to the DSP one, but I'm not that this will always work.